Gruntfile.js

```diff
@@ -541,6 +554,7 @@ module.exports = function(grunt) {
         'scsslint',
         'replace',
         'sass:dev',
+        'sass:lexicon',
```

Comment:
You may want to also add this to the watch:css task

Reply:
Fixed and added it to the scsslint task too